src-vue/src/utils/SortOptions.d.ts

```diff
@@ -0,0 +1,8 @@
+export enum SortOptions {
+    NAME_ASC = 'By name (A to Z)',
+    NAME_DESC = 'By name (Z to A)',
+    DATE_ASC = 'By date (from oldest)',
+    DATE_DESC = 'By date (from newest)',
+    MOST_DOWNLOADED = "Most downloaded",
+    TOP_RATED = "Top rated"
+}
```

Comment:
Missing newline >:(

Reply:
Addressed in c7eba636f434157024c6a69638a0a26b51c4aad4